pontoon/base/models.py

```diff
@@ -1321,6 +1321,16 @@ class Repository(models.Model):
         and the following bits in the URL with `{locale_code}`.
     """)
 
+    api_endpoint = models.CharField("API Endpoint", max_length=2000, blank=True, help_text="""
+        A URL pattern to get repository metadata from. Used during sync for faster retrieval of
+        latest commit hashes in combination with `latest_commit_path`.
+        Supports `{locale_code}` wildcard.
+    """)
+    latest_commit_path = models.CharField(max_length=2000, blank=True, help_text="""
+        A slash-separated list of keys and indices used to retrieve the latest commit hash value
+        from the `api_endpoint` JSON response.
+    """)
```

Comment:
PMs should not bother setting those up. That's why I'm hiding these fields from the currently used and documented project admin interface. I can also hide them from the Django admin.

But I'm not exactly sure if we can automate this.